app/src/main/res/layout/item_selection_interaction_items.xml

```diff
@@ -0,0 +1,31 @@
+<layout xmlns:android="http://schemas.android.com/apk/res/android"
+        xmlns:app="http://schemas.android.com/apk/res-auto">
+  <data>
+    <variable
+      name="viewModel"
+      type="org.oppia.app.player.state.itemviewmodel.SelectionInteractionContentViewModel"/>
+  </data>
+  <RelativeLayout
+    android:id="@+id/checkbox_container"
+    android:layout_width="match_parent"
+    android:layout_height="wrap_content"
+    android:layout_margin="4dp"
+    android:orientation="vertical">
+    <androidx.appcompat.widget.AppCompatCheckBox
```

Comment:
Can this textview be directly a part of `AppCompatCheckBox` ? using `text` attribute

Reply:
Same here. Not actually, We need to pass textview for HTML Parser.